doc/user_guide/user_guide.md

```diff
@@ -90,47 +90,70 @@ The first path element is always the name of the bucket. The next elements are t
 
 #### Password Protected Bucket Access
 
-While buckets can be public for reading depending on their configuration, writing is always password protected. For write operations like copying files to the BucketFS or deleting files from the BucketFS BFSC will retrieve the required write password. BFSC supports to read the password either from an [environment variable](#environment-variables-for-default-parameters) or from an interactive prompt hiding the characters typed by the user.
+While buckets can be public for reading depending on their configuration, writing is always password protected. For write operations like copying files to the BucketFS or deleting files from the BucketFS BFSC will retrieve the required write password. BFSC supports to read the password either from the profile in the [configuration file](#configuration-file) or from an interactive prompt hiding the characters typed by the user.
 
-BFSC does not support to provide the password on the command line to avoid the password showing up in the command history. As a general rule you should never put any credentials directly in to a command line.
+BFSC does not support to provide the password on the command line to avoid the password showing up in the command history. As a general rule you should never put any credentials directly into a command line.
 
-#### Environment Variables for Default Parameters
+#### Retrieving the Password, Base64 Encoding
 
-BFSC supports the following environment variables for applying parameters to all subsequent commands. If an environment variable is unset then BFSC uses the corresponding default value shown in the table below. Parameters supplied on the commandline will override the environment variables.
+The password for write operations to the BucketFS is usually stored in file `/exa/etc/EXAConf`:
+```
+WritePass = <value>
+```
 
-| Parameter                       | Environment variable | Default value |
-|---------------------------------|----------------------|---------------|
-| Host address of BucketFS server | `BUCKETFS_HOST`      | `localhost`   |
-| Port                            | `BUCKETFS_PORT`      | `2580`        |
-| Write passsword                 | `BUCKETFS_PASSWORD`  | (none)        |
-| Name of root bucket             | `BUCKETFS_BUCKET`    | `default`     |
+Additionally the `<value>` is base64 encoded.  For additional convenience BFSC allows you to provide the password in base64 encoded format and let BFSC decode it with commandline flag `--decode-base64-password` or `-d`. This applies to all methods providing the password: via the profile in your configuration file, as well as via interactive prompt.
```

Comment:
This flag should be specified in the profile, not on the command line.

Reply:
OK